mantidproject/mantid
# Update the concepts page on FittingMinimizers

Adds new comparisons: 
- The "Trust region" minimizer
- Tables with results for neutron data and "CUTEst" problems.

**To test:**
- Build the documentation (Qt help or HTML) and check the concep page `FittingMinimizers`
- Check that the new tables make sense. Feedback needed!
- Make sure that there are no details missing.

(Find the problem/data files zip that is used by the scripts here (at ISIS): \\olympic\Babylon5\Public\fedemp\fitting\FittingTestData.zip)

Fixes #17594.



---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
